Assets/MixedRealityToolkit.SDK/Features/Utilities/StabilizationPlaneModifier.cs

```diff
@@ -0,0 +1,390 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+using UnityEngine.XR.WSA;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    [Serializable]
+    public struct StabilizationPlaneOverride
+    {
+        public Vector3 Center;
+        public Vector3 Normal;
+    }
+
+    /// <summary>
+    /// StabilizationPlaneModifier handles the setting of the stabilization plane in several ways.
+    /// Using StabilizationPlaneModifier with override DepthLSR which is automatically enabled via depth buffer sharing
+    /// StabilizationPlaneModifier is recommended for HoloLens 1, can be used for HoloLens 2, and does a no op for WMR
+    /// </summary>
+    public class StabilizationPlaneModifier : MonoBehaviour
+    {
+        [System.Serializable]
+        public enum StabilizationPlaneMode
+        {
+            Fixed,
+            TargetOverride,
+            PlaneOverride,
+            GazeHit,
+            Off
```

Comment:
Nit: It would be good to have comments here for what they mean, so that for code readers/programmatic access folks, they'll have it in their autocomplete/IDE. I think that you have already written these things out in the tooltip below and also in the PR description, so copying those here should suffice